libparsec/crates/testbed/src/template/crc_hash.rs

```diff
@@ -466,11 +466,12 @@ impl CrcHash for BlockAccess {
         digest.crc_hash(hasher);
     }
 }
-impl CrcHash for Chunk {
+impl CrcHash for ChunkView {
     fn crc_hash(&self, hasher: &mut crc32fast::Hasher) {
+        // TODO: change to `ChunkView``
         hasher.update(b"Chunk");
```

Comment:
Do it

![](https://static.wikia.nocookie.net/star-wars-memes/images/6/6d/Dew-it-galactic-republic.gif/revision/latest/scale-to-width-down/300?cb=20211018230421)

Reply:
That would break the testbed server tho, what do you think?

Reply:
Not a problem, we could rebuild it

Reply:
[1a7577b](https://github.com/Scille/parsec-cloud/pull/7698/commits/1a7577b6c81c43428ebdaa3dd74675c832eb2073)